src/js/components/LoginPage/Login.sass

```diff
@@ -1,4 +1,4 @@
-.Login
+.Login form
```

Comment:
please choose a naming convention on your css classes and stick with it. Most commonly used is either camelCase or dash-separated